tests.nix

```diff
@@ -0,0 +1,26 @@
+{ self, pkgs, garnix-lib }: {
+  x86_64-linux.default = pkgs.testers.runNixOSTest ({ lib, ... }: {
```

Comment:
Maybe call this something more informative than `default`? Or is there any advantage in having it be `default`?